bot/exts/filters/token_remover.py

```diff
@@ -176,6 +217,22 @@ def is_valid_timestamp(b64_content: str) -> bool:
             log.debug(f"Invalid token timestamp '{b64_content}': smaller than Discord epoch")
             return False
 
+    @staticmethod
+    def is_maybevalid_hmac(b64_content: str) -> bool:
+        """
+        Determine if a given hmac portion of a token is potentially valid.
+
+        If the HMAC has 3 or less characters, it's probably a dummy value like "xxxxxxxxxx",
+        and thus the token can probably be skipped.
+        """
+        unique = len(collections.Counter(b64_content.lower()).keys())
```

Comment:
This is simpler and doesn't require an import (don't forget to remove the import). 

```suggestion
        unique = len(set(b64_content.lower()))
```

Reply:
That's definitely a better way to do it. Not sure why I didn't catch that >.>